spikeextractors/cacherecordingextractor.py

```diff
@@ -0,0 +1,22 @@
+from spikeextractors.extractors.bindatrecordingextractor import BinDatRecordingExtractor
+import tempfile
+import os
+
+
+class CacheRecordingExtractor(BinDatRecordingExtractor):
+
+    extractor_name = 'CacheRecordingExtractor'
+
+    def __init__(self, recording, dtype=None, chunksize=None):
+        self._tmp_file = tempfile.NamedTemporaryFile(suffix=".dat").name
+        if dtype is None:
```

Comment:
I think we should not have the possibility to change this dtype. It should be always the same as the input.

Reply:
ok! I'll remove that. I'll also pass chunk_size